attributes/default.rb

```diff
@@ -1,37 +1,16 @@
 #
-# Cookbook Name:: maven
-# Attributes:: default
+# Cookbook: maven
+# License: Apache 2.0
 #
-# Author:: Seth Chisamore (<schisamo@chef.io>)
-# Author:: Bryan W. Berry (<bryan.berry@gmail.com>)
-#
-# Copyright:: Copyright (c) 2010-2013, Chef Software, Inc.
-# License:: Apache License, Version 2.0
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+# Copyright 2010-2013, Chef Software, Inc.
+# Copyright 2015, Bloomberg Finance L.P.
```

Comment:
Sure, though I’d defer to this cookbook’s maintainers. I just happened to be swinging by & figured I’d offer my two cents (esp. since I recognized your name).